code/controllers/configuration/entries/game_tweaks.dm

```diff
@@ -19,3 +19,7 @@
 /datum/config_entry/number/brig_timer_preset_long
 	config_entry_value = 5
 	min_val = 1
+
+/datum/config_entry/number/cooldown_antag_time
+	config_entry_value = 3000
+	min_val = 1
```

Comment:
3000 would be 3000 minutes.
Default value should be 0.
Minimum value should be 0 (disabled).